packages/core/src/stringifyPolicy.js

```diff
@@ -0,0 +1,21 @@
+const jsonStringify = require('json-stable-stringify')
+
+module.exports = {
+  jsonStringifySortedPolicy(policy) {
+    return jsonStringify(policy, {
+      space: 2,
+      cmp: diffFriendlyPolicyKeysComparator,
+    })
+  },
+}
+
+function diffFriendlyPolicyKeysComparator({ key: a }, { key: b }) {
+  if (a === b) return 0
+  if (a.includes('>')) {
+    a = a.split('>').reverse().join('<')
+  }
+  if (b.includes('>')) {
+    b = b.split('>').reverse().join('<')
+  }
+  return a < b ? -1 : 1
```

Comment:
inlining the reversing and using `<` instead of localeCompare for performance. No noticable performance gain from avoiding the split.reverse.join (I tested)
Also, localeCompare could work differently on different machines, which we also don't want.

Reply:
on 1000 iterations of sorting metamask policy I get from 7s to 5s with the optimizations and the ifs are the main source of speedup a lot of sorting happens on globals also and they don't have arrows